libevm/VMFace.h

```diff
@@ -25,14 +25,30 @@ namespace dev
 namespace eth
 {
 
-struct VMException: virtual Exception {};
-#define ETH_SIMPLE_EXCEPTION_VM(X) struct X: virtual VMException { const char* what() const noexcept override { return #X; } }
+struct VMException: Exception {};
```

Comment:
This is Boost suggestion: http://www.boost.org/doc/libs/1_63_0/libs/exception/doc/using_virtual_inheritance_in_exception_types.html